packages/governance/src/binaryBallotCounter.js

```diff
@@ -0,0 +1,169 @@
+// @ts-check
+
+import { assert, details as X } from '@agoric/assert';
+import { makeStore } from '@agoric/store';
+import { makePromiseKit } from '@agoric/promise-kit';
+import { Far } from '@agoric/marshal';
+
+import { ChoiceMethod, buildBallot } from './ballotBuilder';
+
+const makeWeightedBallot = (ballot, shares) => ({ ballot, shares });
+
+const makeBinaryBallot = (question, positionAName, positionBName) => {
+  const positions = [];
+  assert.typeof(question, 'string');
+  assert.typeof(positionAName, 'string');
+  assert.typeof(positionBName, 'string');
+  positions.push(positionAName, positionBName);
+
+  return buildBallot(ChoiceMethod.CHOOSE_N, question, positions, 1n);
```

Comment:
It looks like we're still splitting up positions into positionAName and positionBName, and then re-making an array called positions. Why?